EXPENSE_POLICY.md

```diff
@@ -0,0 +1,105 @@
+# Expense Policy
+
+Fastify is an OpenJS Foundation project, and it accepts donations through the
+[Open Collective](https://opencollective.com/fastify/) platform to enhance the
+project and support the community.
```

Comment:
This paragraph is in contradiction with the following one. We are not _accepting donations_ as Fastify, but as the Fastify collaborators.

Reply:
Fixed